Source/WebKitLegacy/mac/WebView/WebUIDelegate.h

```diff
@@ -73,6 +73,9 @@ enum {
     WebMenuItemPDFZoomOut,
     WebMenuItemPDFAutoSize,
     WebMenuItemPDFSinglePage,
+    WebMenuItemPDFSinglePageContinuous,
+    WebMenuItemPDFTwoPages,
+    WebMenuItemPDFTwoPagesContinuous,
```

Comment:
(Ditto, re: ordering of these new SPI enum values)

I'm also curious if we could just omit these as SPI-exposed values, altogether.

Reply:
I can do either of those. Is there a preference to which path we should take?

CC @pxlcoder

Reply:
> I can do either of those. Is there a preference to which path we should take?
> 
> CC @pxlcoder

I think it would be more ideal to avoid adding more legacy SPI, if possible.

Reply:
Agreed, note that this enum is marked as deprecated: `WEBKIT_ENUM_DEPRECATED_MAC`.

Reply:
`toTag` inside of `WebKitLegacy/Mac/WebView/WebHTMLView.mm` seems to be branching on `ContextMenuAction` values and is returning the appropriate enum from `WebUIDelegate` (which is deprecated as Aditya mentioned). Should we keep these values then or is there a specific value we can return for these cases?

Reply:
Can use the `return std::nullopt;` near the bottom.

Reply:
I removed the mentioned enum values and also added a default case to return std:nullopt inside of `toTag`